spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -49,6 +51,27 @@
   end
 
   describe "#publish" do
+    context "2i approval" do
+      it "cannot be published when step by step is not approved" do
```

Comment:
Current wording could be misinterpreted as "cannot be published if changes requested" (the antithesis to "approved"), but what we really mean is "cannot be published if in the wrong state" (`draft`, `submitted_for_2i`, etc).

Might I suggest...?

```ruby
      it "cannot be published when step by step is not in `approved_2i` state" do
```